util/duration_test.go

```diff
@@ -26,10 +29,32 @@ var durationTest = []struct {
 }
```

Comment:
Why do you say it is a breaking change if integers still working?

Reply:
You are probably looking at the test for a different function. The tests for parsing the user-provided strings from `--ttl` to are in `parseDurationTest` and `TestParseDuration`.